link-hint.el

```diff
@@ -1076,6 +1078,23 @@ Only search the range between just after the point and BOUND."
   :at-point-p #'xref--item-at-point
   :copy #'link-hint--copy-xref-item)
 
+;; ** Dictionary button
+(defun link-hint--next-dictionary-button (&optional end-bound start-bound)
+  (let ((pos (next-single-char-property-change (or start-bound (point))
+                                               'button
+                                               nil
+                                               end-bound)))
+    (when (< pos end-bound)
+      (if-let ((button (button-at pos)))
```

Comment:
Link-hint currently supports Emacs 24.4, so don't use if-let (requires >=Emacs 25.1).

Reply:
I was able to delete this function entirely.